jepler_udecimal/utrig.py

```diff
@@ -198,7 +198,13 @@ def asin(x, context=None):
 
     with localcontext(context) as ctx:
         ctx.prec += 2
-        r = atan(x / (1 - x * x).sqrt())
+        pi = Decimal(1).atan() * 4
+        if x == 1:
+            r = pi / 2  # pi * 1/2 radians
+        elif x == -1:
+            r = pi * 3 / 2  # pi * 3/2 radians
```

Comment:
Shouldn't it be -pi/2?

Reply:
Using the unit circle method, 3/2*pi rad is equal to -1/2*pi rad

Reply:
I'd still rather see it match math.asin:
```python
>>> math.asin(-1)
-1.5707963267948966
```

Reply:
Good idea. Hadn't considered compatibility with `math.asin`. Thanks for your patience.